### PR TITLE
(stoa0114a)_all_files

### PR DESCRIPTION
**Pull request by AlisonBabeu**
Updated all the cts_.xml files
Updated all of the TEI headers and bibliographic data

### CODE REVIEW

File: data/stoa0114a/stoa003/stoa0114a.stoa003.opp-lat1.xml [hunk 1]
```diff
@@ -375,7 +367,7 @@
 </div>
 
 <div n="4" subtype="poem" type="textpart">
-<ab> <title>IIII.  EPITHALAMIVM DICTVM MAXIMO V. S. PRAEFATIO. </title> </ab>
+<ab> <title >IIII.  EPITHALAMIVM DICTVM MAXIMO V. S. PRAEFATIO. </title> </ab>
```
Comment (lcerrato) [inline data/stoa0114a/stoa003/stoa0114a.stoa003.opp-lat1.xml:370]: A space seems to have crept into every `<title> ` tag

**Review by lcerrato — COMMENTED** [review]
Weird space crept into `<title>` tag in one file

**Review by lcerrato — CHANGES_REQUESTED** [review]
see comment

File: data/stoa0114a/stoa003/stoa0114a.stoa003.opp-lat1.xml (hunk 1 above)
Comment (AlisonBabeu) [reply data/stoa0114a/stoa003/stoa0114a.stoa003.opp-lat1.xml:370]: Ah, that was my fault, from tryng to add something automatically and then not having looked through whole file. I will go fix that.

Comment (AlisonBabeu) [reply data/stoa0114a/stoa003/stoa0114a.stoa003.opp-lat1.xml:370]: And was that for all of the files @lcerrato or just this one?

Comment (AlisonBabeu) [reply data/stoa0114a/stoa003/stoa0114a.stoa003.opp-lat1.xml:370]: Actually I did a find across the files it was in two files. I've fixed it and will push the changes.

Comment (AlisonBabeu) [reply data/stoa0114a/stoa003/stoa0114a.stoa003.opp-lat1.xml]: hey @lcerrato I fixed the last issue with the space in the title element so can I merge this? Thanks!

**Review by lcerrato — APPROVED** [review]

**Pull request merged**